src/components/note/__internal__/note.style.js

```diff
@@ -0,0 +1,152 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import baseTheme from '../../../style/themes/base';
+
+const NoteContent = styled.div`
+  position: relative;
+  flex-grow: 0;
+  width: 100%;
+  
+  ${({ theme }) => `
+    &:not(:last-of-type) {
+      padding-bottom: 24px;
+    }
+
+    div.DraftEditor-root {
+      min-height: inherit;
+      height: 100%;
+    }
+
+    div.DraftEditor-editorContainer,
+    div.public-DraftEditor-content {
+      min-height: inherit;
+      height: 100%;
+      background-color: ${theme.colors.white};
+      line-height: 21px;
+
+      .public-DraftStyleDefault-ol {
+        position: relative;
+        left: -26px;
+        top: -16px;
+        bottom: 4px;
+        
+        .public-DraftStyleDefault-orderedListItem {
+          .public-DraftStyleDefault-block {
+            padding-left: 4px;
+          }
+        }
+        :last-of-type {
+          margin-block-end: 0px;
+        }
+      }
+
+      .public-DraftStyleDefault-ul {
+        position: relative;
+        left: -22px;
+        top: -12px;
+        bottom: 4px;
+        :last-of-type {
+          margin-block-end: 0px;
+        }
+      }
+    }
+
+    & + & {
+      border-top: solid 1px ${theme.tile.separator};
+    }
+  `}
+`;
+
+const InlineControl = styled.div`
+  position: absolute;
+  top: 24px;
+  right: 16px; 
+  z-index: 100
+`;
+
+const Title = styled.div`
```

Comment:
Seems strange that title is a `div` and not a header, is there a reason for that?

Reply:
Yeah good point, the Title was originally just a container but I went a different way and didn't update the styles